skimage/registration/tests/test_phase_cross_correlation.py

```diff
@@ -263,6 +263,7 @@ def test_disambiguate_zero_shift():
     """
     image = camera()
     computed_shift, _, _ = phase_cross_correlation(
-            image, image, disambiguate=True, return_error='always'
+            image, image, disambiguate=disambiguate, return_error='always'
             )
-    assert computed_shift == (0, 0)
+    np.testing.assert_array_equal(computed_shift, np.array((0., 0.)))
+    assert isinstance(computed_shift, np.ndarray)
```

Comment:
```suggestion
    assert isinstance(computed_shift, np.ndarray)
    np.testing.assert_array_equal(computed_shift, np.array((0., 0.)))
```
This is minor, but it makes more sense in this order.

Reply:
@decorouz feel free to commit if you agree with the change :slightly_smiling_face: